app/config/wallabag.yml

```diff
@@ -64,11 +64,11 @@ wallabag_core:
             section: entry
         -
             name: shaarli_url
-            value: http://myshaarli.com
+            value: https://github.com/shaarli/Shaarli
             section: entry
         -
             name: scuttle_url
-            value: http://scuttle.org
+            value: https://github.com/scronide/scuttle
```

Comment:
Nope, this should be a Scuttle instance URL.

Reply:
please excuse this faulty edits